eole/models/model.py

```diff
@@ -936,7 +952,8 @@ def forward(self, src, tgt, src_len, bptt=False, with_align=False):
         if self.add_estimator:
             # Version with average
```

Comment:
Is this code intended to be used at some point? If so, maybe add some `estimator_mode` flag, else remove?

Reply:
standby no decision for now